tests/test_admin_actions.py

```diff
@@ -647,6 +725,7 @@ def test_signal_when_published(self):
             self.assertEquals(published_mr[0], self.moderation_request1)
             self.assertEquals(kwargs['workflow'], self.collection.workflow)
 
+    # TODO Andrew, how is it intended that collection status should change?
```

Comment:
Ie, right now once all moderation requests are approved the collection status is changed to archived. When reading the tests, it looks like this should NOT be the case. 

Either way, I would suggest that changing how `collection.status` works is out of scope for this PR and it should be changed in new PR.

Reply:
I think this is part of the Versioning state machine where versions state change are forced to go through state changes.

Reply:
The versioning state machine and the collection state machine (albeit simpler than versioning) are pretty much separate. 

The behaviour of the collection state machine is basically the following. 
```
COLLECTING = "COLLECTING"
IN_REVIEW = "IN_REVIEW"
ARCHIVED = "ARCHIVED"
CANCELLED = "CANCELLED"
```

Where there seems to a be bug is that as soon as all requests has been approved the collection state changes from `in review` to `archived`. An archived state suggests that no more actions needs to be taken. Even though the collection is now archived there are still actions to be taken. 

I'll see if I manage to make a separate PR around that. It's out of scope of this PR anyway.